packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -36,6 +41,8 @@ class TerraService {
       // Hides the blinking input caret that appears in inputs or editable text areas.
       hideInputCaret,
 
+      serviceOptions: this.serviceOptions,
```

Comment:
I think we can rework how these options get applied in the utility function. Rather than binding these service options to the global environment we can just call the function from the service with the options. I'll work on a branch with an example to make this more clear using the `accessibility` helper and axe options.